format.go

```diff
@@ -55,19 +55,28 @@ func Unpack(err error) UnpackedError {
 	return upErr
 }
 
-// ToString returns a default formatted string for a given eris error.
-func (upErr *UnpackedError) ToString(format Format) string {
+// ToFormattedString returns a custom formatted string for a given eris error.
+//
+// To declare custom format, the Format object has to be passed as an argument.
+// An error without trace will be formatted as <Root error msg>[Format.ErrorSep]<Wrap error msg>
+// An error with trace will be formatted as following
+// <Root error msg>[Format.MsgStackSep]
+// [Format.PreStackSep]<Method1>[Format.StackElemSep]<File1>[Format.StackElemSep]<Line1>[Format.ErrorSep]
```

Comment:
should we also show method 2 under root error since it'll be added to the stack?